src/library/librarycontrol.cpp

```diff
@@ -482,14 +482,25 @@ void LibraryControl::emitKeyEvent(QKeyEvent&& event) {
     VERIFY_OR_DEBUG_ASSERT(m_pLibraryWidget) {
         return;
     }
-    if (!m_pLibraryWidget->hasFocus() && !m_pSidebarWidget->hasFocus()) {
+    if (!QApplication::focusWindow()) {
+        qDebug()<<"Mixxx window is not focused, don't send key events";
+        return;
+    }
+
+    bool keyIsTab = event.key() == 16777217;
```

Comment:
```suggestion
    bool keyIsTab = event.key() == static_cast<int>(Qt::Key_Tab);
```

Reply:
Alright, thx. Just learned that static_cast is converted at compile time.